modules/search-pipeline-common/src/main/java/org/opensearch/search/pipeline/common/ScriptRequestProcessor.java

```diff
@@ -141,11 +144,24 @@ public Factory(ScriptService scriptService) {
             this.scriptService = scriptService;
         }
 
+        /**
+         * Creates a new instance of {@link ScriptRequestProcessor}.
+         *
+         * @param registry The registry of processor factories.
+         * @param processorTag The processor's tag.
+         * @param description The processor's description.
+         * @param ignoreFailure  option to ignore failure
+         * @param config The configuration options for the processor.
+         * @param pipelineContext The pipelineContext for the processor.
+         * @return The created {@link ScriptRequestProcessor} instance.
+         * @throws Exception if an error occurs during the creation process.
+         */
```

Comment:
Do we need this JavaDoc? The method is inherited from the parent. We generally don't need to write JavaDoc for overridden methods.

Reply:
removed